src/main/java/com/intel/gkl/compression/IntelDeflater.java

```diff
@@ -55,12 +61,14 @@
     public synchronized boolean load(File tempDir) {
 
```

Comment:
Can add some check for tempDir before passing to Native lib loader

Reply:
Same as above - addressed in PR #122